Modules/_operator.c

```diff
@@ -1230,6 +1233,242 @@ static PyType_Spec itemgetter_type_spec = {
     .slots = itemgetter_type_slots,
 };
 
+/* itemtuplegetter object **********************************************************/
+
+typedef struct {
+    PyObject_HEAD
+    PyObject *items;
+    PyObject *defaults;
+    Py_ssize_t nitems;
```

Comment:
getting a number of items in a tuple is very cheap in `C`, thus it is best to minimize class attributes, by not storing `nitems` and `ndefaults`.

Reply:
Hmmm.
It is ~2.5 ns, to be precise.
I see now that `itemgetter` does it, so maybe it makes sense having it.